src/timeline.js

```diff
@@ -462,7 +462,18 @@ export default class Timeline extends React.Component {
      *
      * @type {(tableWidth: number) => void}
      */
-    onTableResize: PropTypes.func
+    onTableResize: PropTypes.func,
+
+    /**
+     * This property controls if the overlapping segments are displayed overlapped
+     * or if they are displayed one beneath the other.
+     *
+     * In this last case, the row expands its height to include all its child segments
+     *
+     * @default false
+     * @type { boolean }
+     */
+    allowOverlappingSegments: PropTypes.bool
```

Comment:
Momentan, e alba sau neagra. Ar trebui sa poata fi si gri. Adica poate vreau sa activez doar pe anumite tipuri de segmente. 

Vin cu urmatoarea propunere. 
* inlaturam aceasta setare globala
* as pune in `ItemRenderer.allowOverlapping`. Ar fi usor sa te legi de ea? Sa obtii rendererul si sa faci treaba? Daca e greu, o alta varianta ar fi ca in locul prop asta, sa avem o functie, care sa primeasca ceva parametru. Ar fi deci posibila customizare. Insa ar fi un mecanism diferit fata de celelalte, unde lucram la nivel de renderer.
* in 99% din cazuri va fi folosita in modul global. Adica asa cum era variabila de mai sus. Asta o putem face folosind `itemRendererDefaultProps`. Corect?
* ca orice proprietate a ItemRenderer, e duala. E si in props si functie. Functie daca cumva are omul un ItemRenderer custom, si are acolo logica care decide. OK?
* am uitat: la noi in ItemRenderer, daca dam in model o proprietate care e in renderer, e copiata automat? Adica ar putea omul sa aiba tot asa o logica selectiva, dar atunci cand genereaza datele?

Reply:
UPDATE cf. discutie:

`displayOnNewRowIfOverlap: boolean | (item) => boolean`

Daca avem la indemana grup: dam si grup in semnatura: group, item